docs/api-docs/store-management/shipping/use-real-time-carrier.md

```diff
@@ -0,0 +1,375 @@
+# Use a Real-Time Carrier
+
+Real-time carriers allow merchants and shoppers to obtain **real-time** shipping quotes. There are two types of real-time carriers: native integrations and shipping provider apps. Shipping provider apps require merchants to install the app on their store. Once a merchant installs a shipping provider app on their store, API users and merchants can connect the carrier to the store. API users and merchants can then define and enable a shipping method for a carrier in one or more shipping zones. They can define and enable a different shipping method in each shipping zone for the same carrier. Enabling a zone's real-time shipping method makes the method available to shoppers in that zone during checkout. 
+
+This article is a guide for API users on how to manage carrier connections and real-time shipping methods from carriers using the [Shipping V2 API](/api-reference/store-management/shipping-api).
+
+## Prerequisites
+- For shipping provider apps only: Merchant has installed the shipping provider app. 
+- You have obtained the required [OAuth Scopes](/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes): `Information & Settings`.
```

Comment:
you don't obtain OAuth scopes; you create or use API accounts configured with those scopes. Review the new auth docs (the two in the `getting started` section of the /docs/ toc) to get more familiar with the concept and language.

think about who these prerequisites are for. you might want to divide this into two sections, one for provider app developers and one for merchants. I'm pretty sure provder app devs will use app API accounts and merchants will use store API accounts. They also have different questions and concerns as they read.

Reply:
The doc is for people who are using the REST management APIs, so i made the prerequisites reflect this (store API account).